src/test/scala/org/clulab/timenorm/scate/EvaluateLinker.scala

```diff
@@ -36,6 +38,10 @@ object EvaluateLinker {
     distances.toSeq.groupBy(_._1).mapValues(_.map(_._2).groupBy(identity).mapValues(_.length).toMap).toMap
   }
 
+  private def textContent(path: String): String = {
+    Source.fromInputStream(this.getClass.getResourceAsStream(path)).getLines.mkString
+  }
+
```

Comment:
Is there any reason you can't just use `Files.readAllBytes(path)` here? We're not actually reading from the classpath in this file are we?